src/main/groovy/life/qbic/datamodel/datasets/NfCorePipelineResult.groovy

```diff
@@ -170,19 +171,30 @@ final class NfCorePipelineResult {
         String name = fileTree.get("name")
         String fileType = fileTree.get("fileType")
         String path = fileTree.get("path")
+        boolean nfCoreFileTypeFound = false
         for (String nfCoreFileType : nfCoreFileTypes) {
             Class<?> c = Class.forName(nfCoreFileType)
             Method method = c.getDeclaredMethod("create", String.class, String.class)
             try {
                 DataFile dataFile = method.invoke(null, name, path) as DataFile
+                nfCoreFileTypeFound = true
                 return dataFile
             } catch (InvocationTargetException e) {
                 // Do nothing as we need to try out all specialisations that extend the
                 // DataFile class
+
             }
         }
+        // We have to check for files of unknown type since this Parser will encounter variable file output dependent on the pipeline
+        if(fileType && !nfCoreFileTypeFound)
+        {
+            //Unknown Files will be ignored for now
+        }
         // If we cannot create a DataFile object at all, throw an exception
+        else
+        {
         throw new IllegalArgumentException("File $name with path $path is of unknown nfcore file type.")
+        }
```

Comment:
```suggestion
        // We have to check for files of unknown type since this Parser will encounter variable file output dependent on the pipeline
        if(!fileType)
        {
        throw new IllegalArgumentException("File $name with path $path is of unknown nfcore file type.")
        }
```